www/src/pages/menu-button.mdx

```diff
@@ -12,10 +12,10 @@ import Helmet from "react-helmet"
 
 # MenuButton (Dropdown)
 
-[Menu](#menu) - 
```

Comment:
I *think* this trailing whitespace needs to be there so that there's a space between the characters.